src/app/core/pusher/pusher.service.ts

```diff
@@ -330,7 +330,25 @@ export class PusherService implements OnDestroy {
       if (this.isCurrentWorkspace(data)) {
         this.store$.dispatch(
           new DocumentsAction.Get({
-            query: {stems: [{collectionId: data.id}], page: 0, pageSize: 100},
+            query: {stems: [{collectionId: data.id}], page: 0, pageSize: 200},
+          })
+        );
+      }
+    });
+    this.channel.bind('Collection:reload', data => {
```

Comment:
Why we send push notification with whole object? I think it's not needed in this case.

Reply:
I used what was already there not to have to create way too much code. Do you think it is worth fixing?

Reply:
Maybe it's not worth it.